docs/customization.md

```diff
@@ -239,12 +239,38 @@ directly in the source of the theme and recompile it.
 
 ### Environment setup
 
-First, clone the repository:
+First, clone the repository for the edition you want to work on. Note
+that you need to be a member of [Insiders] to access its repository.
 
-```
-git clone https://github.com/squidfunk/mkdocs-material
-cd mkdocs-material
-```
+  [Insiders]: insiders/index.md
+
+=== "Material for MkDocs"
+
+    ```
+    git clone https://github.com/squidfunk/mkdocs-material
+    cd mkdocs-material
+    ```
+
+=== "Insiders (with token)"
+
+    You will need to have a GitHub access token [as described in the
+    Insiders documentation] and make it available in the `$GH_TOKEN` 
+    variable. 
+
+    ```
+    git clone https://${GH_TOKEN}@github.com/squidfunk/mkdocs-material-insiders.git
+    ```
+
+    [as described in the Insiders documentation]: insiders/getting-started.md#requirements
+
+=== "Insiders (with SSH key)"
```

Comment:
We're nowhere mentioning cloning with SSH in our docs, so could we move this into a code annotation and remove the "(with token)"? From my experience giving too many options to choose from leads to confusion. Let's recommend the token approach as the canonical way, as most users are used to that, since we advertise it on our docs. I just try to make less work for us 😅